articles/protocols/oauth2/oauth-state.md

```diff
@@ -1,24 +1,26 @@
 ---
-title: The State Parameter
+title: State Parameter
 description: Explains how to use the state parameter in authentication requests to help prevent CSRF attacks.
 ---
 
-# The State Parameter
+# State Parameter
 
 The `state` parameter is an authentication parameter used to help mitigate [CSRF attacks](https://en.wikipedia.org/wiki/Cross-site_request_forgery).
 
-A CSRF attack can occur when a malicious program causes a user's web browser to perform an unwanted action on a trusted site that the user is currently authenticated. This type of attack specifically target state-changing requests to initiate a type of action instead of getting user data because the attacker has no way to see the response of the forged request.
+A CSRF attack, also known as a one-click attack or session-riding, can occur when a malicious program causes a user's web browser to perform an unwanted action on a trusted site upon which the user is currently authenticated. This type of attack targets state-changing requests as opposed to user data to initiate an action.
 
-For the most basic cases the `state` parameter should be a [nonce](https://en.wikipedia.org/wiki/Cryptographic_nonce) as shown in the example below.  **But this field can also be a [Base64](https://en.wikipedia.org/wiki/Base64) encoded json object that can hold multiple values [such as a return URL](/tutorials/redirecting-users).**
+![](/media/articles/protocols/CSRF_Diagram.png)
 
-## How to use the state parameter
+::: note
+Depending on the application type or framework, this may be included for the developer. Also the exact structure of the requests may differ.
+:::
 
-By using the state parameter to hold a value for verification, malicious requests can be denied.
+## How to use the `state` parameter
 
-![](/media/articles/protocols/CSRF_Diagram.png)
+Deny malicious requests by setting the `state` parameter to hold a value for verification.
 
-::: note
-Depending on the application type or framework this may be included for the developer. Also the exact structure of the requests may differ.
+::: Note
```

Comment:
I deployed locally to test the doc (the heroku review app refuses to work) and this displays like that:

![image](https://user-images.githubusercontent.com/11715799/39806860-966a654c-5383-11e8-9544-d6bb1b34ac2a.png)

Looks like the `::: note` syntax is case sensitive. Change and see if it gets fixed? (spoiler alert: it does :P)